lib/nexmo_markdown_renderer/filters/code_filter.rb

```diff
@@ -1,56 +1,60 @@
-class CodeFilter < Banzai::Filter
-  def call(input)
-    input.gsub(/(?!.*snippet)```code(.+?)```/m) do |_s|
-      config = YAML.safe_load($1)
-
-      if config['config']
-        configs = YAML.load_file("#{Rails.root}/config/code_examples.yml")
-        config = config['config'].split('.').inject(configs) { |h, k| h[k] }
+module Nexmo
+  module Markdown
+    class CodeFilter < Banzai::Filter
+      def call(input)
+        input.gsub(/(?!.*snippet)```code(.+?)```/m) do |_s|
+          config = YAML.safe_load($1)
+    
+          if config['config']
+            configs = YAML.load_file("#{Rails.root}/config/code_examples.yml")
+            config = config['config'].split('.').inject(configs) { |h, k| h[k] }
+          end
+    
+          code = File.read("#{Rails.root}/#{config['source']}")
```

Comment:
I think we should not rely on the gem being part of a Rails application, instead, we should use an `ENV` variable that points to the docs and use that as the `root_path`

Reply:
👍